src/js/tabs/servos.js

```diff
@@ -57,9 +57,9 @@ TABS.servos.initialize = function (callback) {
         $('div.tab-servos table.fields tr.main').append(servoHeader);
 
         function process_servos(name, alternate, obj) {
```

Comment:
`alternate` is unused as well.

Reply:
Good catch. Fixed in a new commit. With this commit I have fixed too some others Sonar complaints in the same file... to not upload it alone :)